### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by AngeBleu77** [Meth962/SymphoniaSaveEdit#13] [Q&A]
Hi, i'm a newbie so i don't know how it's working, very sorry...
Could you explain me how to use this mod please, i can't see any .exe file
Thanks in advance ^^

**Comment by SimonStAmand** [comment]
It is not a mod per se, but a tool. There are unfortunately no .exe so you have to build to project yourself in order to use it. You need to download the git project and then a IDE that can build C# project like CLion or Visual Studio (not Visual Studio Code). Open the git projet with said IDE and then you should be able to build it after fidling around a bit (that's the hard part for a newbie). It is kinda easy if you are in the craft, but daunting and foreign if you are not.

**Reply by AngeBleu77** [reply]
thanks for you answer, i really apprecied the time you use it ^^
I will try how to use CLio or VS, i think i could be find some tutorial on Youtube
I have to learn haha
if i have any question, can i ask you for help ?

**Reply by SimonStAmand** [reply]
CLion is great. JetBrain make great IDE. I'll try to give a hand sure

**Comment by AngeBleu77** [comment]
![Error](https://github.com/user-attachments/assets/2a64ec58-d4a3-411e-b78c-16a39298e3a9)

So many error that i don't understand

**Reply by SimonStAmand** [reply]
https://stackoverflow.com/questions/40416569/newtonsoft-could-not-be-found

A nice step by step to install the package can be found here. Googling is the go-to problem solving tool to any programmer.  Most of the time, it gets the job done.

**Reply by AngeBleu77** [reply]
![Nice!](https://github.com/user-attachments/assets/823a3bfc-eec4-4364-86d5-f9439404eeb1)

Big Thanks Bro! ^^

**Reply by Meth962** [reply]
Hrm, were you meaning to build the source code yourself? I think a lot of people aren't seeing there is a "Releases" tab that has the application already in EXE format
https://github.com/Meth962/SymphoniaSaveEdit/releases

Maybe it is hard to see. I've never liked Github's format for that. If you go to "<> Code" tab at the top, on the right-hand side pane there is an easily missed "Releases" link.

**Reply by SimonStAmand** [reply]
After all these time using it! Thanks for poiting that out.

**Comment by The1Azreen** [comment]
Just want to check for ps3 once you save its in a .bin format the one in my ps3 save file is a .BIN is there something I am doing wrong to get save file corruption

**Reply by Meth962** [reply]
Are you also loading the other two files with the .BIN? For PS3 it needs three files, otherwise it can't decrypt the .bin file.

**Comment by Ghostkitten2** [comment]
![Screenshot (107)](https://github.com/user-attachments/assets/976e16a8-b4b3-4423-9d38-2134b57013e4)

On a similar note, I've got it uploaded to CLion, but I'm not sure how I can actually run it, could anyone help?

**Reply by Meth962** [reply]
Do you mean to compile the code on your own or do you just want the app (EXE file)? The https://github.com/Meth962/SymphoniaSaveEdit/releases page has the app to download so you don't have to do all the hard work.
